migrations/1528395626_create_indexes_on_event_logs.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+CREATE INDEX IF NOT EXISTS event_logs_source ON event_logs(source);
+CREATE INDEX IF NOT EXISTS event_logs_date_trunc_timestamp ON event_logs(DATE_TRUNC('day', timestamp));
```

Comment:
nit:
```suggestion
CREATE INDEX IF NOT EXISTS event_logs_date_trunc_day_timestamp ON event_logs(DATE_TRUNC('day', timestamp));
```

Not sure if it's more clear or the current version is sufficient.